### PR TITLE
Add LogChimp to EasyPanel!

### PR DESCRIPTION
**Pull request by DrMxrcy**
**Note:** The image only has a working "latest" tag. When I get around to it, I can build a versioned one.

### CODE REVIEW

File: templates/logchimp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: LogChimp
+description: Build better products with customer feedback.
+instructions: null;
```
Comment (deiucanta) [inline templates/logchimp/meta.yaml:3]: I'm not sure if this is a correct value for instructions. Leave it empty if there are no instructions.

Comment (DrMxrcy) [reply templates/logchimp/meta.yaml]: Whoops, sorry. Put that there by accident!

**Comment by DrMxrcy** [comment]
Closed due to old image, and a lack of an official image.

**Comment by Supernova3339** [comment]
> Closed due to old image, and a lack of an official image.
ghcr.io/logchimp/logchimp
no versioning so unofficial is fine

**Pull request closed without merging**